features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP.cpp

```diff
@@ -39,6 +39,10 @@ static const intptr_t cellular_properties[AT_CellularDevice::PROPERTY_MAX] = {
     0,  // PROPERTY_NON_IP_PDP_TYPE
     1,  // PROPERTY_AT_CGEREP
     1,  // PROPERTY_AT_COPS_FALLBACK_AUTO
+    0,  // PROPERTY_SOCKET_COUNT
```

Comment:
Here for this target `PROPERTY_SOCKET_COUNT` is 7, `PROPERTY_IP_TCP` and `PROPERTY_IP_UDP` is 1.

Reply:
The definitions here are for the modem's onboard IP stack. Since this driver has no `UBLOX_PPP_CellularStack.cpp`, it should be used only with LWIP that has own definitions.